lib/samples/animate_3d_graphic/animate_3d_graphic.dart

```diff
@@ -0,0 +1,610 @@
+                    value: _cameraHeading,
+                    min: -180,
+                    max: 180,
+                    onChanged: (value) {
+                      setModalState(() => _cameraHeading = value);
+                      _cameraController.cameraHeadingOffset = value;
+                    },
+                  ),
+                  _buildSlider(
+                    label: 'Pitch Offset',
+                    value: _cameraPitch,
+                    min: 0,
+                    max: 180,
+                    onChanged: (value) {
+                      setModalState(() => _cameraPitch = value);
+                      _cameraController.cameraPitchOffset = value;
+                    },
+                  ),
+                  SwitchListTile(
+                    title: const Text('Auto Heading'),
+                    value: _autoHeading,
+                    onChanged: (value) {
+                      setModalState(() => _autoHeading = value);
+                      _cameraController.isAutoHeadingEnabled = value;
+                    },
+                  ),
+                  SwitchListTile(
+                    title: const Text('Auto Pitch'),
+                    value: _autoPitch,
+                    onChanged: (value) {
+                      setModalState(() => _autoPitch = value);
+                      _cameraController.isAutoPitchEnabled = value;
+                    },
+                  ),
+                  SwitchListTile(
+                    title: const Text('Auto Roll'),
+                    value: _autoRoll,
+                    onChanged: (value) {
+                      setModalState(() => _autoRoll = value);
+                      _cameraController.isAutoRollEnabled = value;
+                    },
+                  ),
+                ],
+              ),
+            );
+          },
+        );
+      },
+    );
+  }
+
+  // Builds a labeled slider widget.
+  Widget _buildSlider({
+    required String label,
+    required double value,
+    required double min,
+    required double max,
+    required ValueChanged<double> onChanged,
+  }) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text('$label: ${value.toStringAsFixed(0)}'),
+        Slider(
+          value: value,
+          min: min,
+          max: max,
+          divisions: (max - min).toInt(),
+          label: value.toStringAsFixed(0),
+          onChanged: onChanged,
+        ),
+        const SizedBox(height: 8),
+      ],
+    );
+  }
+}
+
+// An enum of the different mission selections available in this sample.
+enum Mission {
+  grandCanyon('Grand Canyon', '290f0c571c394461a8b58b6775d0bd63'),
+  hawaii('Hawaii', 'e87c154fb9c2487f999143df5b08e9b1'),
+  pyrenees('Pyrenees', '5a9b60cee9ba41e79640a06bcdf8084d'),
+  snowdon('Snowdon', '12509ffdc684437f8f2656b0129d2c13');
+
+  const Mission(this.label, this.itemId);
+
+  final String label;
+  final String itemId;
```

Comment:
`label` and `itemId` should have comments to describe what they are. Especially `itemId`, because it's just random hex digits.